tools/osbuilder/scripts/lib.sh

```diff
@@ -349,6 +355,7 @@ RUN ln -sf /usr/bin/g++ /bin/musl-g++
 			-e "s|@INSTALL_MUSL@|${install_musl//$'\n'/\\n}|g" \
 			-e "s|@INSTALL_RUST@|${install_rust//$'\n'/\\n}|g" \
 			-e "s|@SET_PROXY@|${set_proxy:-}|g" \
+			-e "s|@INSTALL_AA_KBC@|${install_aa_kbc//$'\n'/\\n}|g" \
```

Comment:
Can't this just be

```suggestion
			-e "s|@INSTALL_AA_KBC@|${AA_KBC_EXTRAS:-}|g" \
```

and you wouldn't need the block above?